src/zjs_callbacks.c

```diff
@@ -86,6 +92,8 @@ void zjs_init_callbacks(void)
         }
         memset(cb_map, 0, size);
     }
+    zjs_port_ring_buf_init(&ring_buffer, 1024, (uint32_t*)args_buffer);
```

Comment:
Why did you switch from the DECLARE_POW2 method? That seemed sort of safer to ensure we get optimized access, in case Joe Maintenance Programmer alters things. :)